plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/internal/crc/server/core/listeners/ConfigureCRCFrameworksListener.java

```diff
@@ -35,7 +35,7 @@
 
 public class ConfigureCRCFrameworksListener extends UnitedServerListener {
 
-	private static final String CRC_HOST = "https://api.crc.testing";
+	private static final String CRC_HOST_URL = "https://console-openshift-console.apps-crc.testing";
```

Comment:
I don't understand this change ?

Reply:
this url is what you get when you retrieve it via the crc cmd line:
```
$ crc console
```

Reply:
reverted it, given that the REST endpoint is not identical to the console in OS4